### ISSUE TITLE
simple return starts installation process

### ISSUE THREAD

**Issue by mabod** [Morganamilo/paru#42]
When I do:

```
40# paru polkit-d
1 aur/polkit-dumb-agent-git r15.bf4bef0-1 [+0 ~0.00]
    A polkit agent in 145 lines of code using Qt and libsystemd, because polkit sucks.
2 aur/polkit-duktape 0.118-2 [+0 ~0.00]
    polkit with duktape as the javascript engine
:: Packages to install (eg: 1 2 3, 1-3 or ^4): 
```

paru is prompting me for my choice. If I do not make a selection but simply press enter without giving a number paru starts the installation:

```
40# paru polkit-d
1 aur/polkit-dumb-agent-git r15.bf4bef0-1 [+0 ~0.00]
    A polkit agent in 145 lines of code using Qt and libsystemd, because polkit sucks.
2 aur/polkit-duktape 0.118-2 [+0 ~0.00]
    polkit with duktape as the javascript engine
:: Packages to install (eg: 1 2 3, 1-3 or ^4): 
:: Resolving dependencies...
:: Calculating conflicts...
:: Calculating inner conflicts...

:: Conflicts found:
    polkit-duktape: polkit  

:: Conflicting packages will have to be confirmed manually

Repo (4) libutempter-1.2.1-1  kpty-5.75.0-1  kdesu-5.75.0-1  duktape-2.6.0-1
Aur (2) polkit-dumb-agent-git-r15.bf4bef0-1  polkit-duktape-0.118-2

:: Proceed to review? [Y/n]: 
```

While yaru stops the process if you just hit enter without selecting a specific number.

I like the  yaru behaviour better. It is more logical. I did not make a selection so why would paru install something?

**Comment by ashhaxie** [comment]
Can you please let us know which version of paru you're using? paru is designed this way more from a point from the upgrade menu, it was overlooked, but I believe this was fixed in commit ba00b9f so you may not be using the most up to date version.

**Comment by Morganamilo** [comment]
Duplicate of #35
